README.md

```diff
@@ -30,9 +30,11 @@ If you run into a bug when using librespot, please search the existing issues be
 # Building
 Rust 1.32.0 or later is required to build librespot.
 
-We recently switched to using [Rodio](https://github.com/tomaka/rodio) for audio playback by default, hence for macOS and Windows, you should just be able to clone and build librespot (with the command below). For linux, you will need to run the additional commands below, depending on your distro.
+## Additional Dependencies
+We recently switched to using [Rodio](https://github.com/tomaka/rodio) for audio playback by default, hence for macOS and Windows, you should just be able to clone and build librespot (with the command below).
+For Linux, you will need to run the additional commands below, depending on your distro.
 
-On debian / ubuntu, the following command will install these dependencies :
+On Debian/Ubuntu, the following command will install these dependencies :
 ```shell
 sudo apt-get install build-essential libasound2-dev
```

Comment:
Isn't `libasound2-dev` only required for the ALSA backend?

Reply:
Rodio(Cpal) [as well](https://github.com/RustAudio/cpal/blob/8f28048f611dacb7d6ef2df4f26d951d6511f670/.github/workflows/cpal.yml#L12)?